src/libraries/System.Text.Json/src/Polyfills/System.Numerics.BitOperations.netstandard20.cs

```diff
@@ -0,0 +1,64 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// Contains a polyfill implementation of System.Numerics.BitOperations that works on netstandard2.0.
```

Comment:
Should this (and any other polyfills) be moved the `Common` folder perhaps?

Reply:
Removed the copies and referenced CoreLib directly.